src/Parser/Warlock/Destruction/Modules/Talents/FireAndBrimstone.js

```diff
@@ -1,69 +0,0 @@
-import React from 'react';
-
-import Analyzer from 'Parser/Core/Analyzer';
-import Combatants from 'Parser/Core/Modules/Combatants';
-
-import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
-import SpellIcon from 'common/SpellIcon';
-import { formatNumber, formatPercentage } from 'common/format';
-import SPELLS from 'common/SPELLS';
-
-import SoulShardEvents from '../SoulShards/SoulShardEvents';
-
-class FireAndBrimstone extends Analyzer {
```

Comment:
Can you explain why this module should be deleted?

Reply:
takes up too much space for what it is, if you really wanted to keep it for “comparison” reasons highly suggest small box, looks bad as it was